src/assets/i18n/en.json

```diff
@@ -131,5 +131,15 @@
     "From": "From",
     "Message": "Message",
     "btn-send": "Send"
+  },
+  "events": {
+    "title": "Events and Activities",
+    "description": ["Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque a felis tortor. Etiam laoreet felis mi,",
```

Comment:
I'm not sure why this is an array, could you explain me please? :)

Reply:
this is to add multiple lines

Reply:
Shouldn't the text just be adjusted according to the size? Or are you separating this in this way for multiple paragraphs? If this is to separate paragraphs maybe this makes sense. Otherwise I'm not sure why this couldn't be just a string. I will pull your changes and review this locally to understand it better.